src/client/include/com/amazonaws/kinesis/video/client/Include.h

```diff
@@ -928,6 +941,12 @@ typedef enum {
     CONTENT_STORE_PRESSURE_POLICY_DROP_TAIL_ITEM = 1,
 } CONTENT_STORE_PRESSURE_POLICY;
 
+// KVS Stream events (bit flags)
+#define STREAM_EVENT_TYPE_IMAGE_GENERATION 0x00000001L
+#define STREAM_EVENT_TYPE_NOTIFICATION     0x00000002L
+// used to iterative purposes, always keep last.
+#define STREAM_EVENT_TYPE_LAST 0x00000004L
+

```

Comment:
Sorry for the confusion but instead of making this complete untyped I think we should have it still have an event type just with those values, same as how we do for the frame flags, see for example:  

https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/master/src/mkvgen/include/com/amazonaws/kinesis/video/mkvgen/Include.h#L176-L201